packages/relay/src/formatters.ts

```diff
@@ -232,6 +237,36 @@ const stringToHex = (str) => {
   return hex;
 };
 
+const formatHex = (hexValue: string): string => (hexValue === '0x' ? '0x0' : hexValue);
+
+const convertToHex = (value: number | string | null): string => {
+  // Check for null value
+  if (value === null || value === undefined) {
```

Comment:
since `value` only accepts number|string|null, value === undefined would not make sense.

Reply:
good point @quiet-node, 

anyways, the simples way to eval for both `null`, `undefined` and `empty string` would be:

```
if (!value) {...
```

Reply:
Removed `value === undefined`